tests/__init__.py

```diff
@@ -20,4 +20,4 @@
 # flake8: noqa
 
 from .api import *  # type: ignore
-from .core import *  # type: ignore
+from .test_core import *  # type: ignore
```

Comment:
Do we still need this line? It was I think only a kludge as our files weren't named as expected, but it now is

Reply:
It seems that everything can be removed from this `__init__` ;)